src/NodeosBlock.ts

```diff
@@ -36,7 +36,7 @@ export class NodeosBlock implements Block {
     }))
   }
 
-  private flattenArray(arr: any[]): any[] {
+  protected flattenArray(arr: any[]): any[] {
```

Comment:
Reason for changing to protected?

Reply:
Ah because I had initially tried to have MongoBlock inherit from NodeosBlock but that was more trouble than it was worth and forgot to change this back.